src/components/modals/SupportRequestModal.js

```diff
@@ -0,0 +1,382 @@
+          if (response.status === 201) {
+            Notifications.showSuccess(
+              {text: `Sent Successfully`, layout: 'topRight', timeout: 1500}
+            );
+            await this.setState(prev => {
+              prev.type = 'question';
+              prev.subject = '';
+              prev.description = '';
+              prev.attachment = '';
+              return prev;
+            });
+            this.props.onOKRequest('support');
+          } else {
+            Notifications.showError({
+              text: `ERROR ${response.status} : Unable To Send`,
+              layout: 'topRight',
+            });
+          }
+        }
+      };
+
+      this.nameChangeHandler = (e) => {
+        const nameText = e.target.value;
+        this.setState(prev => {
+          prev.name = nameText;
+          return prev;
+        });
+      };
+
+      this.typeChangeHandler = (e) => {
+        const typeText = e.target.value;
+        this.setState(prev => {
+          prev.type = typeText;
+          return prev;
+        });
+      };
+
+      this.subjectChangeHandler = (e) => {
+        const subjectText = e.target.value;
+        this.setState(prev => {
+          prev.subject = subjectText;
+          return prev;
+        });
+      };
+
+      this.descriptionChangeHandler = (e) => {
+        const descriptionText = e.target.value;
+        this.setState(prev => {
+          prev.description = descriptionText;
+          return prev;
+        });
+      };
+
+      this.attachmentChangeHandler = (e) => {
+        this.setState(prev => {
+          prev.attachment = e;
+          return prev;
+        });
+      };
+
+      this.attachmentCancel = () => {
+        this.setState(prev => {
+          prev.attachment = '';
+          return prev;
+        });
+      };
+
+      this.emailChangeHandler = (e) => {
+        let emailText = e.target.value;
+        this.setState(prev => {
+          prev.email = emailText;
+          prev.valid = /.+@.+\.[A-Za-z]+$/.test(emailText) ? true : false;
+          return prev;
+        });
+      };
+
+      this.handleResize = () => {
+        this.setState(prev => {
+          prev.height =  Storage.userIsLogged() ? (window.innerHeight < 550 ? window.innerHeight : '550px') : (window.innerHeight < 700 ? window.innerHeight : '700px');
+          return prev;
+        });
+      };
+    }
+
+    render() {
+      window.addEventListener('resize', this.handleResize);
+
+      const customStyles = {
+        overlay: {
+          position: 'fixed',
+          top: this.state.top,
+          left: '10',
+          right: '0',
+          bottom: '0',
+          backgroundColor: 'static',
+          width: '400px',
+          height: this.state.height,
+          padding: '0px 450px 700px 0px',
+        },
+
+        content: {
+          position: 'fixed',
+          top: '1',
+          left: '1',
+          right: '0',
+          bottom: '0',
```

Comment:
Here, and in the overlay, I would suggest setting right and bottom at 20px. That sets the widget off the edges enough to look more like a widget.